### PR TITLE
[FLINK-27722] Add Slack community information and invite link

### PR DESCRIPTION
**Pull request by MartijnVisser**
@xintongsong If you have a Chinese translation, I could also directly add the Chinese translation to the page.

### CODE REVIEW

File: community.md
Comment (rmetzger) [inline community.md:149]: I wonder if it makes sense to use the ASF URL shortener: https://s.apache.org/
It allows updating the underlying link (however only for the owner of the id) .. but it would still be easier than merging a change to flink-web after 100 invites.

File: community.md
Comment (rmetzger) [inline community.md]:
```suggestion
  "If it didn’t happen on a mailing list, it didn’t happen." - [The Apache Mottos](http://theapacheway.com/on-list/)
```

File: community.md
Comment (rmetzger) [inline community.md]:
```suggestion
Due to Slack limitations the invite link expires after 100 invites. If it is expired, please reach out to the [Dev mailing list](#mailing-Lists). 
```
Maybe it makes sense to make this more explicit, so that people understand why this happens?

**Review by rmetzger — APPROVED** [review]
Thanks a lot for adding Slack to the website!

File: community.md
Comment (xintongsong) [reply community.md]: +1

File: community.md
Comment (xintongsong) [inline community.md]: We need an empty line here, otherwise the rules are shown as a paragraph rather than a list.
```suggestion
There are a couple of community rules: 

* **Be respectful** - This is the most important rule!
```

File: community.md
Comment (xintongsong) [reply community.md:149]: From my side, I'd be more concerned if the invite link can only be updated by one person, compared to having to change the website once a while.

Comment (rmetzger) [reply community.md:149]: If that person fails to update the link, we can still update the website.
And that person should be one of the three folks involved in this PR ;)

Comment (xintongsong) [inline community.md:149]: The invite link is not the only way to join the workspace. Every existing member is free to invite new members, without needing approval from admins. We may want to mention and even suggest that. Hopefully that reduces the frequency we have to update the invite link.

**Comment by xintongsong** [comment]
> @xintongsong If you have a Chinese translation, I could also directly add the Chinese translation to the page.

Sure. Let's address the comments for the English version first, and I'll add a translation before merging.

File: community.md
Comment (MartijnVisser) [reply community.md:149]: I agree with @xintongsong that I don't think that having one person who can update the shortened URL is a good idea. I rather just update the Slack invite link (for now) in the project website, since any committer can fix that.

Comment (rmetzger) [reply community.md:149]: Allright 👍🏻

Comment (MartijnVisser) [reply community.md:149]: Good point, I've added a comment for that.

**Review by xintongsong — APPROVED** [review]
Thanks @MartijnVisser. LGTM.

I've made a few minor changes (see the fixup commit) and added the Chinese translation.

File: community.md
Comment (wuchong) [reply community.md:149]: Shall we add a note here for how to reach out to the community that the invitation link hit the limitation?

**Pull request merged**
